docs/docs/reference/principled-meta-programming.md

```diff
@@ -378,9 +384,9 @@ aspect is also important for macro expansion.  To illustrate this,
 consider an implementation of the `power` function that makes use of a
 statically known exponent:
 
-    inline def power(inline n: Int, x: Double) = ~powerCode(n, '(x))
+    inline def power(inline n: Int, x: Double): Double = ~powerCode(n, '(x))
 
-    private def powerCode(n: Int, x: Expr[Double]): Expr[Double] =
+    private def powerCode(n: Int, x: Expr[Double]): Staged[Double] =
```

Comment:
Here `Staged` is just an implicit function type `Staged[X] = implicit Staging => Expr[X]` to avoid an explicit implicit in the signature. We could find another name for `Staging`/`Staged`.